<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInDialog</class>
 <widget class="QDialog" name="SignInDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MidasClient Sign in</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DionysusIcons.qrc">
    <normaloff>:/icons/Icon16.PNG</normaloff>:/icons/Icon16.PNG</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../DionysusIcons.qrc">:/icons/dude3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Profile:</string>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="../DionysusIcons.qrc">:/icons/earth_network.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Server URL:</string>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="serverEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="editServerSettingsButton">
       <property name="toolTip">
        <string>Edit Server Settings</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../DionysusIcons.qrc">
         <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="profileComboBox"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="createProfileButton">
       <property name="toolTip">
        <string>Create a Profile</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../DionysusIcons.qrc">
         <normaloff>:/icons/add2.png</normaloff>:/icons/add2.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>profileComboBox</tabstop>
  <tabstop>createProfileButton</tabstop>
  <tabstop>serverEdit</tabstop>
  <tabstop>editServerSettingsButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SignInDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SignInDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
